source/musubi.f90

```diff
@@ -98,21 +112,47 @@ program musubi
   call mus_init_bcTimer( geometry%boundary%nBCtypes )
 
   ! initialize musubi
-  call mus_initialize( scheme     = scheme,     &
-    &                  solverData = solverData, &
-    &                  geometry   = geometry,   &
-    &                  params     = params,     &
-    &                  control    = control     )
+  call mus_initialize( scheme        = scheme,                     &
+    &                  solverData    = solverData,                 &
+    &                  geometry      = geometry,                   &
+    &                  params        = params,                     &
+    &                  control       = control,                    &
+    &                  particle_kind = particleGroup%particle_kind )
+
+  call mus_load_particles( particleGroup = particleGroup, &
+                        & conf = params%general%solver%conf(1), &
+                        & chunkSize = 100,  &
+                        & scheme = scheme, &
+                        & geometry = geometry, &
+                        & myRank = params%general%proc%rank )
```

Comment:
Why are the particles loaded unconditionally, but only initialized if the kind is not `'none'`? It would make sens, if `mus_load_particles` fills in this information, abut `particleGroup%particle_kind` is already used above in `mus_initialize` (where I think it should go into `params` rather than modifying the interface).